bark_ml/python_wrapper/BUILD

```diff
@@ -9,6 +9,7 @@ cc_binary(
     "//bark_ml/commons:spaces",
     # external
     "@python_linux//:python-lib",
+    #"@pytorch_api//:pytorch-lib",
```

Comment:
do we not need that

Reply:
we should not have any commented code (without further comments). Either comment or remove